netcam.c

```diff
@@ -2803,7 +2803,7 @@ int netcam_start(struct context *cnt)
 
     if ((retval = pthread_create(&netcam->thread_id, &handler_attribute,
                                  &netcam_handler_loop, netcam)) < 0) {
-        MOTION_LOG(ALR, TYPE_NETCAM, SHOW_ERRNO, "%s: Starting camera"
+        MOTION_LOG(NTC, TYPE_NETCAM, SHOW_ERRNO, "%s: Starting camera"
```

Comment:
This one is on a failure path (thread creation failing) I think so perhaps should be ALR?

(Perhaps the message should mention 'error' too!)